### DISCUSSION TITLE
"ADT needs to validate subscription details. 0 days remaining." warning encountered

### DISCUSSION THREAD

**Discussion by balaedit** [airsdk/Adobe-Runtime-Support#2732] [General]
Hello, we purchased a license april month (11th april 2023) and all was well and the harman splash screen is removed.
However, while building the APK/aab/debug file this week, we suddenly encountered the warning below : 

![animateCCerror](https://github.com/airsdk/Adobe-Runtime-Support/assets/37571867/dc404088-7963-46ec-8c79-81f484a00afc)

I checked via cmd but same error : 
![cmdError](https://github.com/airsdk/Adobe-Runtime-Support/assets/37571867/b2e2b649-c58c-4f46-a418-f715abec1495)

After this error, license file in AIRSDKmanager is automatically removed and when I import again, it just show the loading bar.
![airSDKerror](https://github.com/airsdk/Adobe-Runtime-Support/assets/37571867/273f8ce2-d212-4d64-9fab-660d1cc7eefd)

Is also check the aws link which was provided on mail, response : 
![awsLink1](https://github.com/airsdk/Adobe-Runtime-Support/assets/37571867/5fb402f7-1c6d-44fc-a844-b6ec930b7f11)

Is there any other way to update/validate the license key via cmd.

@ajwfrost java certification has taken lots of time to resolve the issue but please look into this. It's urgent.

**Comment by ajwfrost** [comment]
So basically ADT can't connect to the server... the license code is valid, per your query using a browser, but the Java runtime isn't able to connect to this.

Likely an SSL or proxy issue, I think we've discussed before but don't recall -> do you use a proxy server? if so you can set an environment variable HTTPS_PROXY with the server name and port for this.

To get you unstuck though, I can send a new file over; but it would also be good to see if you can get that call with the `-licenseDevID` to work..

thanks

**Reply by balaedit** [reply]
I had replaced the adt.lic from the today's mail but same message occurred. Error : License has been removed. Warning - ADT license has expired.
